frontend/src/views/DashboardPage/components/SecretInputRow/SecretInputRow.tsx

```diff
@@ -199,6 +199,15 @@ export const SecretInputRow = memo(
 
     const isCreatedSecret = !secId;
     const shouldBeBlockedInAddOnly = !isCreatedSecret && isAddOnly;
+    
+    const hasEqualSign = (str:string) => str.includes("=");
```

Comment:
can you keep this function outside the component if its just a utility function

Reply:
sure @akhilmhdh